helm/values.yaml

```diff
@@ -175,6 +179,12 @@ corda:
       allowDevCorDapps:
         # enabled is a boolean value for the above parameter
         enabled: true
+  # nodebridge is the options that define how to define the node.conf and bridge.conf files which defines the bridge/node tunnel      
```

Comment:
This section is extra, we already have firewall section below, which should be used instead.